docs/source/examples/login.js

```diff
@@ -37,19 +37,19 @@ if (!session.sessionInfo.isLoggedIn) {
   // Redirect the user to their identity provider:
   // (This moves the user away from the current page.)
   session.login({
-      // The URL of the user's OIDC issuer
+      // The URL of the user's OIDC issuer. Specify your OIDC issuer URL.
```

Comment:
This seems confusing - as I understand it, the OIDC issuer isn't determined by the developer, but by the user. (e.g. if their Pod is hosted on inrupt.net, they enter inrupt.net, if it's hosted on solid.community, they enter solid.community.) @NSeydoux is that correct?

(It also seems somewhat redundant with the sentence before it?)

Reply:
Ah - yup.  I'll pop in "Specify " in front of the first sentence.  I just wanted to distinguish between comments where we're explaining what the source is doing and comments where we're pointing out places that users need to update.